### PR TITLE
Improve Readme and add file structure information to make it more open-source friendly

### PR DESCRIPTION
**Pull request by JoshuaPCruz**
### Improve Readme and add file structure information to make it more open-source friendly

# Issue Reference: https://github.com/ayushagg31/Trellis/issues/9

# Changes

- Adding new file for code of conduct
- Adding link to that file on the README
- Adding file structure on README file

# Expected Outcome

- README file updated
- New file on root for code of conduct

# Additional Information

The link on the README file, is pointing to the main repo, just to keep that in mind

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ * Posting or threatening to post other people's personally identifying information ("doxing").
+ * Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+ * Inappropriate photography or recording.
+ * Inappropriate physical contact. You should have someone's consent before touching them.
+ * Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances.
+ * Deliberate intimidation, stalking or following (online or in person).
+ * Advocating for, or encouraging, any of the above behavior.
+ * Sustained disruption of community events, including talks and presentations.
+
+## 5. Weapons Policy
```
Comment (ayushagg31) [inline CODE_OF_CONDUCT.md:46]: Do we really need weapon policy :)

**Comment by ayushagg31** [comment]
This looks good, feel free to move on to other issues. I'll reviewing and updating it soon. Thanks for the contrubution.

**Comment by ayushagg31** [comment]
- Please remove code of conduct entirely, I don't find it relevant to the project
- Also there's some scope of improvement in the project structure, for now add file structure only for client directory, you can use reference from these projects - https://github.com/kriasoft/Folder-Structure-Conventions, https://github.com/oldboyxx/jira_clone/tree/master/api, https://github.com/oldboyxx/jira_clone/tree/master/client
- Add file structure in a README.md inside client file 

Let me know what are your thoughts on this.

**Comment by JoshuaPCruz** [comment]
Ok, looks good, I'll make the change ASAP

**Comment by JoshuaPCruz** [comment]
Hi,
I've already made the changes, let me know if you need me to change
something more

**Comment by ayushagg31** [comment]
Hi JoshuaPCruz, some description in file stucture needs improvement, I'll edit your PR over weekend.

**Review by ayushagg31 — APPROVED** [review]

**Pull request merged**